src/main/controllers/dataset_aggregator.ts

```
@@ -221,9 +221,11 @@ const generateDatasetState = async (state: Record<string, any>) => {
 export default {
     name: 'dataset:state',
     handler: () => async (request: Request, response: Response, next: NextFunction) => {
-        let { datasetId } = request.params;
-        let { status } = request.query;
+        let datasetId = _.get(request, 'params.datasetId');
+        let status = _.get(request, 'query.status');
```

ESLint

[error] 'status' is never reassigned. Use 'const' instead.

(prefer-const)